src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
```

Comment:
PullProofAI

Consider adding a validation decorator for the maximum length of the email field to ensure data integrity.
```diff
+   @MaxLength(255)
```



<!-- This is an auto-generated comment by PullProofAI -->